### DISCUSSION TITLE
FastLogin dont work on 1.8: Invaild session

### DISCUSSION THREAD

**Discussion by GM3322** [games647/FastLogin#794] [Q&A] [answered]
Hello! i have plugin updated and Protocol Lib too but when i try to join it writes: Invalid session (i have original account)
log is here: 
![image](https://user-images.githubusercontent.com/84188600/168157942-0f472abb-3aff-4042-bb40-d755731a5d55.png)

**Comment by TuxCoding** [comment, marked as answer]
Outdated plugin version. This type is more like a bug report btw.
